client/src/pages/EpisodeShowPage/TimerAndTimeBudgetInfo/ReadableDuration.tsx

```diff
@@ -0,0 +1,99 @@
+import { Duration } from "luxon";
+import * as React from "react";
+
+class ReadableDuration extends React.Component<any,  any> {
+  public render() {
+    const durationInMs = Duration.fromMillis(this.props.durationInMs);
+    const duration = durationInMs.shiftTo("days", "hours", "minutes", "seconds");
+
+    const isADayOrMore = duration.days > 0;
+    const isAnHourOrMore = duration.hours > 0;
+    const isAMinuteOrMore = duration.minutes > 0;
+    const isASecondOrMore = duration.seconds > 0;
```

Comment:
This seems similar to code posted above. I think there's a lot of this duration-specific code spread through a bunch of these files. Do you think they could be made into a library file or similar?